write-data.mdx

```diff
@@ -18,7 +22,11 @@ const collectionReference = db.collection("City");
 
 async function createRecord () {
   // .create(args) args array is defined by the constructor fn
-  const recordData = await collectionReference.create(["new-york", "New York"]);
+  const recordData = await collectionReference.create([
+    "new-york", 
+    "New York", 
+    db.collection("Person").record("johnbmahone")
```

Comment:
The returned JSON example below is missing this (value would be `{ "collectionId": "your-namespace/Person", "id": "johnbmahone" }`)